content/en/api/v2/security-monitoring/examples.json

```
@@ -1,4 +1,190 @@
+    }
+  },
+  "DeleteCustomFramework": {
+    "responses": {
+      "200": {
+        "json": {
+          "data": {
+            "attributes": {
+              "description": "this is an example description",
```

```suggestion
              "description": "this is an example description",
```
Small suggestion, should this be filled out with the actual description ?

---

oh the description was "this is an example description" I will try to think of a better example

---

ok thanks for the context there! I think this is probably fine, I was just missing the context.